lib/Db/CardMapper.php

```diff
@@ -236,10 +236,12 @@ public function findAssignedCards($boardId, $username) {
 			->from('deck_cards', 'c')
 			->innerJoin('c', 'deck_stacks', 's', 's.id = c.stack_id')
 			->innerJoin('s', 'deck_boards', 'b', 'b.id = s.board_id')
-			->innerJoin('c', 'deck_assigned_users', 'u', 'c.id = u.card_id')
+			->leftJoin('c', 'deck_assigned_users', 'u', 'c.id = u.card_id')
 			->where($qb->expr()->eq('s.board_id', $qb->createNamedParameter($boardId, IQueryBuilder::PARAM_INT)))
-			->andWhere($qb->expr()->eq('u.participant', $qb->createNamedParameter($username, IQueryBuilder::PARAM_STR)))
-			->andWhere($qb->expr()->eq('u.type', $qb->createNamedParameter(Acl::PERMISSION_TYPE_USER, IQueryBuilder::PARAM_INT)))
+			->andWhere($qb->expr()->orX(
+				$qb->expr()->eq('u.participant', $qb->createNamedParameter($username, IQueryBuilder::PARAM_STR)),
+				$qb->expr()->isNull('u.participant'))
+			)
```

Comment:
As mentioned in the chat, the method should be renamed to also indicate that unassigned cards are fetched as well.